index.html

```diff
@@ -64,7 +64,8 @@ <h2>I'm a Software Developer</h2>
             </div>
         </section>
         <section id="portafolio">
-            <div class="worksCards">
+            <!--<div id="blureContainer" class="blureStyle"></div>-->
+            <div id="cardsContainer" class="worksCards">
                 <div id="WorkDiv" class="worksInit">
                     <div class="WorkTitle">
                         <h2>My Recent Works</h2>
```

Comment:
- [ ] This first card of your project from `line 74` needs to be generated as well dynamically using javascript alongside the other cards.
- [ ] It also does not have the functionality for when the button is clicked to display a popup becasue no JavaScript implementation has been done for it. Please kindly fix both issues as it is important that your app works as expected :+1: 
- > This is the affected section below
![no-open](https://github.com/alexansaa/Personal_Portafolio/assets/89055502/731b3313-a142-450c-82f4-86e1330f8f44)